cmd/sops/main.go

```diff
@@ -415,7 +415,7 @@ func main() {
 		if c.NArg() < 1 {
 			return common.NewExitError("Error: no file specified", codes.NoFileSpecified)
 		}
-		if c.Bool("in-place") && c.String("output") != "" {
+		if c.Bool("in-file") && c.String("output") != "" {
```

Comment:
No, I meant the correct string is `--in-place`, as that's the value of the flag for operating on the file in-place. We don't use `--in-file`.

Reply:
Oh sorry my bad